test/parallel/test-http2-compat-errors.js

```diff
@@ -15,7 +15,7 @@ const server = h2.createServer(common.mustCall(function(req, res) {
   req.on('error', common.mustNotCall());
   res.on('error', common.mustNotCall());
   req.on('aborted', common.mustCall());
-  res.on('aborted', common.mustNotCall());
+  res.on('aborted', common.mustCall());
```

Comment:
Can you confirm that the corresponding code for HTTP1 would actually emit `'aborted'` here?

Reply:
@ronag `ServerResponse` did not emit `'aborted'` with the following code.
```js
// differences from the HTTP/2 corresponding code
// IncomingMessage: did not emit 'error', emitted only 'aborted' event
// ServerResponse: did not emit neither 'error' nor 'aborted' events
const server = http.createServer(common.mustCall((req, res) => {
  // req.on('error', common.mustCall()) // fails
  // res.on('error', common.mustCall()) // fails
  req.on('aborted', common.mustCall(() => {
    assert.ok(req.aborted); // passes
  }));
  // res.on('aborted', common.mustCall()) // fails
  res.write('xyz');
  server.close();
})).listen(0, common.mustCall(function () {
  const request = http.get({
    path: '/',
    port: server.address().port
  }, function(res) {
    res.on('data', common.mustCall((d) => {
      request.destroy();
    }));
  });
}));
```